frontend/server/src/DAO/Tags.php

```diff
@@ -75,4 +75,53 @@ public static function findPublicTagsByPrefix(
 
         return $results;
     }
+
+    /**
+     * @return list<string>
```

Comment:
De acuerdo al comentario de abajo, puedes cambiar la anotación por esta:
```suggestion
     * @return list<array{name: string}>
```